tests/test_coordinated_workers/test_coordinator.py

```diff
@@ -281,3 +284,103 @@ def test_tracing_receivers_urls(
             "otlp_http": "5.6.7.8:4318",
             "otlp_grpc": "5.6.7.8:4317",
         }
+
+
+@pytest.fixture
+def cluster_state():
```

Comment:
since you're not reusing this fixture, and it's not very generic, I'd recommend putting the code directly in the test. 
That or, give it a more descriptive name such as "state_with_cluster_relation_invalid_databag"

Reply:
I have moved it to the test function.